### PR TITLE
Feature/195 trading software

### PR DESCRIPTION
**Pull request by vladimirchr**
#195

### CODE REVIEW

File: HW6/Vladimir_Chirushin/trading/trading-software/Devices/DataBaseDevice.cs [hunk 1]
```diff
@@ -0,0 +1,242 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace trading_software
+{
+    public class DataBaseDevice : IDataBaseDevice
```
Comment (chesdenis) [inline HW6/Vladimir_Chirushin/trading/trading-software/Devices/DataBaseDevice.cs]: Perfect, but would be really cool if you separate this class to stocks, clients ... repositories  with appropriate names.

File: HW6/Vladimir_Chirushin/TradingSoftware/TradingSoftware.ConsoleClient/Program.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TradingSoftware.ConsoleClient
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```
Comment (chesdenis) [inline HW6/Vladimir_Chirushin/TradingSoftware/TradingSoftware.ConsoleClient/Program.cs]: why this main is empty?

File: HW6/Vladimir_Chirushin/TradingSoftware/TradingSoftware.Core.Tests/BlockOfSharesManagerTests.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TradingSoftware.Core.Tests
+{
+    [TestClass]
+    public class BlockOfSharesManagerTests
+    {
+        [TestMethod]
+        public void AddShareTest()
```
Comment (chesdenis) [inline HW6/Vladimir_Chirushin/TradingSoftware/TradingSoftware.Core.Tests/BlockOfSharesManagerTests.cs]: please add tests implementations

**Comment by vladimirchr** [comment]
#Work finished. Ready for review.

**Pull request merged**